<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdersWindow</class>
 <widget class="QDialog" name="OrdersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orders</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="tblCaption">
     <property name="text">
      <string>Orders in DB</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tblOrders">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lbUpdate">
          <property name="text">
           <string>Update Order</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtOrderNum"/>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxPaid">
          <property name="text">
           <string>Mark as Paid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxDelivered">
          <property name="text">
           <string>Mark as delivered</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUpdate">
          <property name="text">
           <string>update</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
